composer.json

```diff
@@ -22,9 +22,9 @@
     ],
     "require": {
         "php": "^5.6.0||^7.0.0",
-        "friendsofsymfony/http-cache": "^2.0.0",
-        "symfony/framework-bundle": "^2.8||^3.0",
-        "symfony/http-foundation": "~2.8.13||^3.1.6"
+        "friendsofsymfony/http-cache": "^2.1.x-dev",
```

Comment:
looking at travis-ci, i think this should be `^2.1@dev` to allow composer to install a dev version. we now tell it to install at least the specific dev version, but not allow to install dev versions (afaik composer does not interpret the .x-dev to understand that it should install a dev version.